Update README.md (#798)
Azure/azure-policy: README.md

May 2020: Microsoft.DocumentDB/databaseAccounts/ipRangeFilter updated from a string property to an array.  Please re-author your custom definitions to support the property as an array.  
July 2020: The alias Microsoft.Sql/servers/securityAlertPolicies/emailAddresses[] and related policies were deprecated. 

> ### AKS Policy does not current support Application gateway Ingress controller and Nginx controller currently 

Reviewer: ```suggestion
### Azure Policy for AKS currently does not support Application Gateway ingress controllers or Nginx controllers
```

Reviewer 2: @nehakulkarni123 - Ingress issue is fixed and deployed to Public, Fairfax, Mooncake clouds now.

Author: excellent! will remove

> The existing ingress policy (i.e. "Kubernetes clusters should be accessible only over HTTPS" ) currently checks only Google ingress controllers and do not currently support AKS ingress controllers. We are working on updating this in the upcoming months to support both controllers. 

Reviewer: ```suggestion
The existing ingress policy (i.e. "Kubernetes clusters should be accessible only over HTTPS" ) currently checks only Google ingress controllers and does not currently support AKS ingress controllers. We are working on updating this in the upcoming months to support both controllers. 
```

### Policy evaluating containers does not support container exclusions currently.  

> Some containers are currently marked as non-compliant without ability to modify, such as Istio init containers. This is because some containers like Istio are loaded as side cars, which thus prevents annotation from being set ahead of time. As a solution for this scenario, we are working on updating policy definitions with an 'excludedContainers' parameter to exclude containers in the constraint template and Azure Policy definition in the upcoming months. 

Reviewer 2: @nehakulkarni123  FYI, fix for this should be available in production by end of August.

Author: Great, added that

May 2020: Microsoft.DocumentDB/databaseAccounts/ipRangeFilter updated from a string property to an array.  Please re-author your custom definitions to support the property as an array.  
July 2020: The alias Microsoft.Sql/servers/securityAlertPolicies/emailAddresses[] and related policies were deprecated. 

> ### Policy evaluating containers does not support container exclusions currently.  

Reviewer 3: This README is a mismash of issues from all types of policies.  This issue needs to make it clear what policy modes this issue pertains to (i.e. just the Kubernetes modes).  Otherwise the reader won't know what the word "container" means

> ### AKS dataplane policies that evaluate containers does not support container exclusions currently.  

Reviewer 3: dataplane is the internal term. Docs should refer to what "Resource Provider Modes" are affected  (https://docs.microsoft.com/en-us/azure/governance/policy/concepts/definition-structure#resource-provider-modes)

Author: good call, updated!
